lib/core/src/lib/app-config/app-config.service.ts

```diff
@@ -70,11 +70,11 @@ export class AppConfigService {
     };
 
     status: Status = Status.INIT;
-    protected onLoadSubject: ReplaySubject<any>;
+    protected onLoadSubject: Subject<any>;
```

Comment:
Reverted the previous change, the `onLoadSubject` does not take into account  extension loading, therefore it can be misleading